### ISSUE TITLE
can't install Prestissimo

### ISSUE THREAD

**Issue by adeguntoro** [hirak/prestissimo#234]
```[InvalidArgumentException]
  Package hirak/prestissimo at version 0.3.8 has a PHP requirement
  incompatible with your PHP version, PHP extensions and Composer version
```
Right now i use Windows 7 Pro SP1, PHP 7.3.9 from XAMPP, and Composer 2.0.2. Is this normal ?

**Comment by leimd** [comment]
Same here, this is breaking our CI builds

**Comment by leimd** [comment]
Actually, if you're getting this error, you are probably running composer v2 now, which means it will download packaged in parallel by default, so that means you don't need this plugin anymore. see here: https://github.com/hirak/prestissimo/issues/230

**Comment by Rakhmanov** [comment]
Yep, Composer changed their latest tag to version 2. That's another reminder not to use latest in your builds.

**Comment by Shleif91** [comment]
> Yep, Composer changed their latest tag to version 2. That's another reminder not to use latest in your builds.
> So if you have this magic line in your docker:
> `COPY --from=composer /usr/bin/composer /usr/bin/composer`
> add `:1` after composer like the following, to keep using version one, until you figure out the impact.
> `COPY --from=composer:1 /usr/bin/composer /usr/bin/composer`

```
FROM composer:1 as vendor
RUN composer global require hirak/prestissimo
```
works, thanks
